sycl/include/CL/sycl/detail/backend_traits_opencl.hpp

```diff
@@ -109,15 +104,28 @@ template <> struct BackendReturn<backend::opencl, device> {
   using type = cl_device_id;
 };
 
+#ifdef SYCL_GET_NATIVE_BACKEND_OPENCL_RETURN_T_CL_EVENT
+template <> struct interop<backend::opencl, event> { using type = cl_event; };
+template <> struct BackendInput<backend::opencl, event> {
+  using type = cl_event;
+};
+template <> struct BackendReturn<backend::opencl, event> {
+  using type = cl_event;
+};
+#else
+template <> struct interop<backend::opencl, event> {
+  using type = std::vector<cl_event>;
+  using value_type = cl_event;
+};
 template <> struct BackendInput<backend::opencl, event> {
   using type = std::vector<cl_event>;
   using value_type = cl_event;
 };
-
```

Comment:
Nit, this is unnecessary change

Reply:
Unchanged, BackendReturn template follows BackendInput with the missing line, but now they are in #ifdef block, where all templates (in my realization) go without the missing line. This deletion is a consequence. Would you prefer, missing lines between templates in #ifdef block or not?